src/python_testing/TC_OCC_3_1.py

```diff
@@ -0,0 +1,115 @@
+#
+#    Copyright (c) 2024 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#

```

Comment:
Need to add the CI boilerplate to get this running in CI. Also add to tests.yaml

Reply:
No CI is planned for this test case.  Only a simple tester manual control is considered to trigger the occupancy sensor on or off.

Reply:
I will try to add CI boilerplate. Where is tests.yaml?

Reply:
Will file an issue for supporting CI.

Reply:
Will triage and resolve in the new issue filed at https://github.com/project-chip/connectedhomeip/issues/34686